support-files/mariadb@.service.in

```diff
@@ -1,29 +1,120 @@
-# Multi instance version of mariadb. For if you run multiple versions at once.
-# Also used for mariadb@bootstrap to bootstrap Galera.
+# Multi instance version of MariaDB
 #
-# create config file @sysconf2dir@/my{instancename}.cnf
+# Use this if you run multiple instances of MariaDB on a single server.
+#
+# Copyright notice:
 #
-# start as systemctl start mariadb@{instancename}.server
-
 # This file is free software; you can redistribute it and/or modify it
 # under the terms of the GNU Lesser General Public License as published by
 # the Free Software Foundation; either version 2.1 of the License, or
 # (at your option) any later version.
 #
-# Thanks to:
-# Daniel Black
-# Erkan Yanar
-# David Strauss
-# and probably others
-# Inspired from https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-db/mysql-init-scripts/files/mysqld_at.service
+# MULTI INSTANCES
+#
+# When multiple instances of MariaDB are running on a server they need to
+# ensure that they don't conflict with each other. This includes elements
+# like network ports, sockets and data directories. The systemd environment
+# variable MYSQLD_MULTI_INSTANCE controls each instance to ensure it is
+# run independently.
+#
+# This is not suitable for Galera as specialised SST recovery scripts are
+# needed.
+#
+# Suffix Mechanism (default):
+#
+# MYSQLD_MULTI_INSTANCE=--defaults-group-suffix=.%I --basedir=@prefix@
+#
+# With this option, the [mysqld.{instancename}] group is read from the default
+# configuration file.
+#
+# Command Line Mechanism:
+#
+# MYSQLD_MULTI_INSTANCE="--socket=/var/run/mysqld/%I.sock \
+#                        --datadir=/var/lib/mysqld-multi/%I \
+#                        --skip-networking"
+#
+# This is a good way run multiple instance where there is little difference
+# in configuration between instances.
+#
+# Configuration File Based Mechanism:
+#
+# MYSQLD_MULTI_INSTANCE=@sysconfdir@/my%I.cnf
```

Comment:
Shouldn't it be?
`MYSQLD_MULTI_INSTANCE="--defaults-file=@sysconfdir@/my%I.cnf"`

Reply:
This seems to be already noticed above (https://github.com/MariaDB/server/pull/510#issuecomment-390801929)